pkg/helper/envVar.go

```diff
@@ -1,8 +1,6 @@
 package helper
 
-import (
-	"k8s.io/kubernetes/pkg/api"
-)
+import "k8s.io/kubernetes/pkg/api"
```

Comment:
This is an unnecessary change.  Did your editor/IDE do this?  I thought it was _best practice_ to use the former approach but I can't find any literature to back it up.  Minor nit...